lib/internal/modules/esm/loader.js

```diff
@@ -96,24 +98,18 @@ class ESMLoader {
    */
   translators = getTranslators();
 
+  /**
+   * Type of loader.
+   * @type {'default' | 'internal'}
+   */
+  loaderType = 'default';
```

Comment:
should this be private?

Reply:
The class and its instances are internal anyway (unless I'm missing something), so no big deal if we want to change that later. It can't be a private field though, or we would need to add a getter, and that seems more trouble that needed for something internal. I'm not particularly proud of this pattern tbh, but also it gets the job done so...